backtesting/backtesting.py

```diff
@@ -1036,10 +1044,10 @@ def __init__(self,
 
         # Convert index to datetime index
         if (not data.index.is_all_dates and
-            not isinstance(data.index, pd.RangeIndex) and
-            # Numeric index with most large numbers
-            (data.index.is_numeric() and
-             (data.index > pd.Timestamp('1975').timestamp()).mean() > .8)):
+                not isinstance(data.index, pd.RangeIndex) and
+                # Numeric index with most large numbers
+                (data.index.is_numeric() and
+                 (data.index > pd.Timestamp('1975').timestamp()).mean() > .8)):
```

Comment:
I'd ask to avoid other code changes (like opinionated reformatting), unrelated to converting comment type annotations, at least within this PR.